capstone-app/src/app/modules/history/wiki-results.service.spec.ts

```diff
@@ -49,6 +41,11 @@ describe('WikiResultsService', () => {
       request.url.includes('https://www.googleapis.com/customsearch/v1')
     );
     expect(req.request.method).toBe('GET');
-    req.flush(testWikiResult);
+    req.flush(testResponse);
+  });
+
+  it('should format the string properly', () => {
+    expect(service.fixString(MockWikiResponse.parse.text['*']))
+    .toEqual(testFixStringResponse);
   });
```

Comment:
Can we add a test to verify the logic in the service parses the raw html correctly?

Reply:
The mock wiki response has the exact format of text that the http call would return, so this is verifying that it is parsing the html correctly.